frontend/src/pages/HomePage/queries/usePostDataForReviewRequestCode.test.tsx

```diff
@@ -0,0 +1,30 @@
+import { renderHook, act, waitFor } from '@testing-library/react';
+
+import QueryClientWrapper from '@/queryTestSetup/QueryClientWrapper';
+
+import { CREATED_REVIEW_REQUEST_CODE } from '../../../mocks/mockData/group';
```

Comment:
경로에 절대 경로 사용하면 어떨까요?
```suggestion
import { CREATED_REVIEW_REQUEST_CODE } from '@/mocks/mockData/group';
```